hermes-2.cxx

```diff
@@ -245,6 +245,14 @@ int Hermes::init(bool restarting) {
                      .doc("Include electron inertia in Ohm's law?")
                      .withDefault<bool>(true);
 
+  ramp_j_diamag = optsc["ramp_j_diamag"]
+                      .doc("ramps up current drive for j_diamag term in vorticity eq")
+                      .withDefault(1.0);
+
+  // Options::root()["mesh"]["paralleltransform"].as<std::string>()
+  // TIMESTEP =  Options::root()["TIMESTEP"].as<BoutReal>() * 1e-6;
+  NOUT =  Options::root()["NOUT"].as<BoutReal>();
```

Comment:
I think this can be removed; NOUT (and TIMESTEP) can be used in BOUT.inp expressions if needed.